utils/UninstallTemplate/Program.cs

```diff
@@ -17,36 +16,49 @@ private enum TemplateNameKind
             Directory
         }
 
-        static void Main(string[] args)
+        static int Main(string[] args)
         {
             if (args.Length < 2)
             {
                 ShowUsage();
-                return;
+                return 1;
             }
 
             var kind = GetTemplateNameKind(args[0]);
             if (kind == TemplateNameKind.None)
             {
                 ShowUsage();
-                return;
+                return 1;
```

Comment:
It's better to use named constants instead of magic numbers

Reply:
Done